packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -30,8 +30,13 @@ const getDefaultConfig = (isDistributable: boolean): ApmAgentConfig => {
     return {
       active: false,
       globalLabels: {},
+      centralConfig: false,
+      captureHeaders: false,
```

Comment:
Could you add a comment explaining why it's disabled?

Reply:
I am erring on the side of caution here and trying to avoid capturing any data that could be sensitive. We could be more selective about this and capture only headers we know to be safe (an allowlist of headers). WDYT?

Reply:
Locked these down in production for now 👍